<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>REIXSXESSpectrometerControlEditor</class>
 <widget class="QGroupBox" name="REIXSXESSpectrometerControlEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectrometer</string>
  </property>
  <property name="title">
   <string>Spectrometer</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labelGrating">
     <property name="text">
      <string>Grating</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="gratingSelectorBox"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="gratingFeedbackLabel">
     <property name="text">
      <string>Currently: LEG</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelEnergy">
     <property name="text">
      <string>Energy</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="energyBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="suffix">
      <string> eV</string>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>80.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2400.000000000000000</double>
     </property>
     <property name="value">
      <double>395.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="energyFeedbackLabel">
     <property name="text">
      <string>Currently: 395 eV</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelDefocus">
     <property name="text">
      <string>Defocus</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="defocusOffsetBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="suffix">
      <string> mm</string>
     </property>
     <property name="minimum">
      <double>-200.000000000000000</double>
     </property>
     <property name="maximum">
      <double>200.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelTiltOffset">
     <property name="text">
      <string>Tilt Offset</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="tiltOffsetBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="suffix">
      <string> deg</string>
     </property>
     <property name="minimum">
      <double>-8.000000000000000</double>
     </property>
     <property name="maximum">
      <double>8.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="energyRangeLabel">
     <property name="text">
      <string>361 eV - 438 eV (78eV)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="stopButton">
     <property name="text">
      <string>Stop!</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/Close.png</normaloff>:/Close.png</iconset>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="moveNowButton">
     <property name="text">
      <string>Move Now</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/system-software-update.png</normaloff>:/system-software-update.png</iconset>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Grating Mask</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="maskComboBox">
     <item>
      <property name="text">
       <string>Pinhole</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mask</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Out</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="maskFeedbackLabel">
     <property name="text">
      <string>Currently: Unknown</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
